src/store/setToolMode.js

```diff
@@ -556,6 +556,7 @@ function _determineStringBindings(toolName, options, interactionTypes) {
   const globalTool = store.state.globalTools[toolName];
 
   if (globalTool) {
+    // eslint-disable-next-line new-cap
```

Comment:
My opinion of this may differ to @dannyrb 's, but here is my argument: It is a class constructor, even if its a reference, so `GlobalTool` should be capitalized, instead of adding a linting exception. There are likely a few other places in the codebase where this is true.

Reply:
Capitalising `GlobalTool` won't fix this. It's the `.tool` property on `globalTool` which is the constructor.

Could rewrite as follows to avoid the linter error.
```js
const globalTool = store.state.globalTools[toolName];
if (globalTool) {
   const { tool: Tool, configuration } = globalTool;
   const tool = new Tool(configuration);

  // ... 
```

The other option would be to change every usage of `globalTools.tool`  to `globalTools.Tool`.

Reply:
I'm okay with your current fix or the workaround. While `Tool` should probably be capitalized, let's file that away mentally as a breaking change for a different day.

Reply:
Ahh good catch, I missed that. I'm happy with how you've done it for now. Its an improvement over what we had before. I'd like to do a sweeping review of constructors in the future though.